sharray.md

```diff
@@ -0,0 +1,90 @@
+# Sharray
+
+> Sharded IPLD Array
+
+The Sharray is a fixed width IPLD tree structure used to store an array of items.
```

Comment:
Just to be clear, this _is_ for fixed sized arrays. O(n) insertion isn't going to be an issue.

Reply:
yeah. This is fixed size arrays, whose items are all known at creation time. Build-once, use many.